src/encoded/types/cell_sample.py

```diff
@@ -0,0 +1,17 @@
+from snovault import collection, load_schema
+
+from .sample import Sample
+
+
+@collection(
+    name="cell-samples",
+    unique_key="submitted_id",
+    properties={
+        "title": "Cell Samples",
+        "description": "Samples consisting of isolated cells",
+    },
+)
+class CellSample(Sample):
```

Comment:
Does this 'inheritance' work as expected?  I ask because last I knew non-abstract items could only inherit from abstract items.  Great if this is no longer the case for SMaHT but wanted to make sure.

Reply:
I believe so, as I think we have non-abstract inheritance in CGAP with WorkflowRun and WorkflowRunAwsem. Sample is an abstract collection here though.